R/wwinference.R

```diff
@@ -121,7 +121,10 @@ wwinference <- function(ww_data,
   if (generate_initial_values) {
     init_lists <- c()
     for (i in 1:mcmc_options$n_chains) {
-      init_lists[[i]] <- get_inits_for_one_chain(stan_args, params)
+      init_lists[[i]] <- get_inits_for_one_chain(
```

Comment:
@kaitejohnson, I think this fixes a bug. Before, it was getting `params` from somewhere I was not sure. Can you validate that what I did is correct?

Reply:
Oh yup that was a bug that probably hasn't gotten caught in the vignette because we defined `params` in the vignette.... 

Also realizing this is out of date we need to merge #58!